example/examples/PriceMarker.js

```diff
@@ -1,17 +1,14 @@
-let React = require('react');
-const ReactNative = require('react-native');
-let {
+import React, { Component } from 'react';
+import {
   StyleSheet,
   View,
   Text,
-} = ReactNative;
+} from 'react-native';
 
-const PriceMarker = React.createClass({
-  getDefaultProps() {
-    return {
+class PriceMarker extends Component {
+  defaultProps = {
```

Comment:
This syntax makes `defaultProps` an instance variable, right?  We want this to be a class property instead of an instance property. We typically follow the pattern:

```js
const defaultProps = {
  foo: 'bar',
};

class MyComponent extends Component { ... }

MyComponent.defaultProps = defaultProps;
```

Check out the Airbnb React styleguide for reference: https://github.com/airbnb/javascript/tree/master/react